service/dap/server_test.go

```diff
@@ -1654,6 +1655,41 @@ func TestSetBreakpoint(t *testing.T) {
 	})
 }
 
+// TestWorkingDir executes to a breakpoint and tests that the specified
+// working directory is the one used to run the program.
+func TestWorkingDir(t *testing.T) {
+	runTest(t, "workdir", func(client *daptest.Client, fixture protest.Fixture) {
+		fixtureWorkingDir, err := filepath.Abs(filepath.Join(protest.FindFixturesDir(), "buildtest"))
+		if err != nil {
+			t.Fatalf("failed to get working directory: %e", err)
+		}
+		runDebugSessionWithBPs(t, client, "launch",
+			// Launch
+			func() {
+				client.LaunchRequestWithArgs(map[string]interface{}{
+					"mode":        "exec",
+					"program":     fixture.Path,
+					"stopOnEntry": false,
+					"cwd":         fixtureWorkingDir,
```

Comment:
What directory is this by default? A comment explaining the diff might be helpful.
Also you might want to match to this similar test:
https://github.com/go-delve/delve/blob/2e80b32c417494ab3fabe929e7051076a7049f7c/pkg/proc/proc_test.go#L2592-L2596

Reply:
Sounds good. Will match to the proc test to make it clearer that this is not the default working dir.